app/controllers/Application.scala

```diff
@@ -111,14 +112,14 @@ class Application(
   def newContributionsLanding(countryCode: String): Action[AnyContent] = maybeAuthenticatedAction().async { implicit request =>
     type Attempt[A] = EitherT[Future, String, A]
     implicit val settings: Settings = settingsProvider.settings()
-    val visitToken = VisitToken(new SecureRandom())
+    val scopeToken = CreateScopeToken(new SecureRandom())
```

Comment:
I decided scopeToken made more sense as this token is used for the scope of access given to the identity id from the user of this page.  Named after where it's going rather than where it came from.  However naming is hard and this isn't instantly understandable .

Reply:
I find the term 'Scope' confusing here, it doesn't really describe the purpose of the token. Do you think we could come up with something that does? Maybe `deDupingToken` or something along those lines?

Reply:
jonny and I have come up with session id, and we are going to store it in the local session storage

Reply:
I like that better, it's a less overloaded term 👍

Reply:
thanks, see new PR https://github.com/guardian/support-models/pull/43